packages/OmegaPrint-Core.package/OPPrinter.class/instance/value..st

```diff
@@ -12,6 +12,5 @@ value: inputNode
 		ifTrue: [ ^ result ]
 		ifFalse: [ | spaces |
 			spaces := self value: aNode skippedSpacesNodes.
-			spaces ifNotEmpty: [ spaces := spaces , self newline.
-				self commentMap add: aNode interval start ].
-			^ spaces , result ]
\ No newline at end of file
+			spaces ifNotEmpty: [ self commentMap add: aNode interval start ].
+			^ self joinValuesSeparatedByNewline: { spaces . result } ]
```

Comment:
warum nicht join: seperatedBy: self newline?

Reply:
in der Annahme dass das haeufiger gerufen wird, habe ich dafuer ne abkuerzung erstellt. Die kann man natuerlich wieder entfernen, wenn die doch nicht so oft genutzt werden